test/recipes/70-test_tls13messages.t

```diff
@@ -296,4 +301,57 @@ checkhandshake($proxy, checkhandshake::HRR_RESUME_HANDSHAKE,
                | checkhandshake::PSK_CLI_EXTENSION
                | checkhandshake::PSK_SRV_EXTENSION,
                "Resumption handshake with HRR test");
+
+#Test 16: signature_algorithms with 1.3-only ClientHello
+my $testtype = PURE_SIGALGS;
+$proxy->clear();
+$proxy->clientflags("-tls1_3");
+$proxy->filter(\&modify_sigalgs_filter);
+$proxy->start();
+
+#Test 17: signature_algorithms with backwards compatible ClientHello
+$testtype = COMPAT_SIGALGS;
+$proxy->clear();
+$proxy->filter(\&modify_sigalgs_filter);
+$proxy->start();
```

Comment:
Oh, that's why I couldn't find the plan that had the number of tests -- it's *in* the Test 1 block.